urdf/sdf_extensions.md

```diff
@@ -0,0 +1,822 @@
+specific elements may do so using the `<gazebo>` tag in the original URDF file.
+This is known as the Gazebo extension to the [URDF
+specification](http://wiki.ros.org/urdf/XML) (see
+http://wiki.ros.org/urdf/XML/Gazebo)
+
+The URDF to SDFormat conversion usually happens automatically without users
+observing the resultang SDFormat file. To diagnose any issues that might come
+up during the conversion, the tool 
+
+```gz sdf -p <path to urdf file>```
+
+can be used to convert the URDF file to SDFormat that can be inspected by the user.
+
+The following is the documentation of the various tags available under
+`<gazebo>` and their corresponding effect on the SDFormat output. While some of
+the content included here is also available in the ["Using a URDF in
+Gazebo"](http://gazebosim.org/tutorials?tut=ros_urdf&cat=connect_ros) tutorial
+of Gazebo-classic, this document should provide a more complete discussion of
+the extension, especially regarding fixed joint lumping.
+
+
+## `<gazebo>` Elements For `<robot>`
+`<gazebo>` tag without a `reference` attribute applies to the SDFormat `<model>`
+that gets generated from the `<robot>` URDF tag. All elements in the `<gazebo>`
+tag are inserted into the SDF `<model>` tag for the generated SDF.
+
+**Example:**
+
+```xml
+<!--URDF-->
+<robot name='no_ref_example'>
+  <link name='world'/>
+  <gazebo>
+    <static>true</static>
+    <plugin name='testPlugin' filename='testFileName'/>
+  </gazebo>
+</robot>
+```
+
+results in:
+
+```xml
+<!--SDFormat-->
+<sdf version='1.9'>
+  <model name='no_ref_example'>
+    <static>true</static>
+    <plugin name='testPlugin' filename='testFileName'/>
+  </model>
+</sdf>
+```
+
+## `<gazebo>` Elements For Links
+
+When using the `<gazebo>` extension for links, the name of the link has to be
+specified in the `reference` attribute. There are a number of special tags that
+modify the values of elements or attributes in the generated SDFormat file. Any
+tag that is not listed in the table below will be directly inserted into the
+the corresponding `<link>` element in the SDFormat output.
+
+**Table of elements with special meaning:**
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Type</th>
+    <th>Description</th>
+    <th>Corresponding SDFormat element</th>
+  </tr>
+  <tr>
+    <td>turnGravityOff</td>
+    <td>bool</td>
+    <td>A value of "true" turns gravity off. Alternatively, <code>gravity</code> (with opposite boolean value) can be used via blob insertion</td>
+    <td><code>gravity</code></td>
+  </tr>
+  <tr>
+    <td>dampingFactor</td>
+    <td>double</td>
+    <td>Exponential velocity decay of the link velocity - takes the value and multiplies the previous link velocity by (1-dampingFactor).</td>
+    <td><code>velocity_decay/linear</code> and <code>velocity_decay/angular</code></td>
+  </tr>
+  <tr>
+    <td>maxVel</td>
+    <td>double</td>
+    <td>maximum contact correction velocity truncation term.</td>
```

Comment:
consider linking to the following gazebo-classic tutorial for the relevant parameters:

https://classic.gazebosim.org/tutorials?tut=physics_params&cat=physics#Contactparameters

Reply:
Good idea! Done in https://github.com/gazebosim/sdf_tutorials/pull/88/commits/3beee03fc26bf4c00d3002e39380d7672c7351ca